GitUI/Script/FilePrompt.Designer.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace GitUI.Script
```

Comment:
Few issues:
![image](https://user-images.githubusercontent.com/4403806/42204620-8bd142d0-7ee5-11e8-86ef-5ce937a361f0.png)

* The form should be vertically restricted - have a look at `FormOpenRepository`.
* The 'Browse' button is misaligned with the textbox.
* Use `FolderBrowserButton` instead of a normal `Button`.
* Labels to be placed to the left of controls, not above.
* Give label a meaningful name instead of `label1`.
* The form must have a correct heading (not `FilePrompt`), something like "Select script files" (open to ideas).
* Control names shouldn't have `_` - either Hungarian notation (e.g. `btnOk`) or Pascal case (e.g. `Ok`). Personally I prefer the Hungarian notation (for control naming), but the source contains a mix. Either is fine.

It is very much preferable if you design the form at 96dpi, otherwise it is hard to avoid visual artifacts at 100% scaling. I'm guessing the misalignment of controls is a result of design at 120dpi.

Reply:
Just saying, this one will probably take me some time, since I am not that strong with WinForms (originally a WPF guy).

Reply:
I've pushed a tweak to the form, please have a look.

Reply:
I approve. Thanks for the help :)

Reply:
It looks like I may have failed to update the forms heading - please fix it when squashing